model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -246,6 +248,8 @@
 NUM_DOWNSTREAM_REQUEST_RETRIES = 80  # has to be high enough so that the retries take the 5 minutes
 DOWNSTREAM_REQUEST_TIMEOUT_SECONDS = 5 * 60  # 5 minutes
 
+LATEST_CONFIG_MAP_NAME = "model-engine-inference-framework-latest-config"
```

Comment:
nit: `LATEST_INFERENCE_FRAMEWORK...`? feels a bit ambiguous as it is currently heh